mist-core/src/main/java/edu/snu/mist/core/MistLauncher.java

```diff
@@ -98,8 +97,6 @@ public static Configuration getDriverConfiguration(final Configuration conf) {
     final JavaConfigurationBuilder jcb = Tang.Factory.getTang().newConfigurationBuilder(conf);
     jcb.bindImplementation(NameResolver.class, LocalNameResolverImpl.class);
     jcb.bindImplementation(IdentifierFactory.class, StringIdentifierFactory.class);
-    jcb.bindConstructor(SpecificResponder.class, SpecificResponderWrapper.class);
```

Comment:
Why is it removed?

Reply:
We don't use specific responder in MistDriver anymore.